draft-davidben-tls-trust-expr.md

```diff
@@ -354,11 +366,11 @@ Each CA regularly fetches trust store manifests from root programs in which it p
 2. For each trust store version in the cached manifest's `versions`:
 
    {: type="a"}
-   1. Look for a trust store entry whose `id`, when indexed into `trust_anchors`, matches the certification path's trust anchor.
+   1. Look for a trust store entry whose `trust_anchor` field, when indexed into `trust_anchors`, matches the certification path's trust anchor.
 
    2. If found, output a TrustStoreInclusion structure:
 
-      * Set `trust_store` to the manifest's `name`, encoded in UTF-8 {{!RFC3629}}, and the version's version number.
+      * Set `trust_store` to the manifest's `id`, encoded in binary form as described in {{trust-store-ids}}, and the version's version number.
```

Comment:
s/binary/der/

Reply:
trust-store-ids refers to it at the binary form. I didn't want to say DER because then we have to clarify whether the tag and length are included. (They're not.) So trust-store-ids defines it once by name and we reference it.

Reply:
Yeah, let's not create more of the dreaded 20 octets fun.